src/app/mbs/module-detail/module-detail.component.html

```diff
@@ -108,6 +108,14 @@ <h2 class="title">Module #{{ module.id }}</h2>
         <td scope="row">Version</td>
         <td scope="row">{{ module.version }}</td>
       </tr>
+      <tr>
+        <td scope="row">MBS JSON</td>
+        <td scope="row">
+          <a *ngIf="module.id" target="_blank" href="{{ getModuleJson(module.id) }}">
```

Comment:
Could you create the string in the `href` directly such as this? If you use a function, then Angular will call the function after every single event (e.g. mouse movements) to ensure the value hasn't changed by that event. Having many of these will reduce the performance of the app and cause unnecessary work by the browser.
```suggestion
          <a *ngIf="module.id" target="_blank" href="{{ mbsUrl + 'module-builds/' + module.id }}">
```